products/BellHybrid/alarms/src/actions/FrontlightAction.cpp

```diff
@@ -2,19 +2,28 @@
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "FrontlightAction.hpp"
+#include "BellAlarmConstants.hpp"
 
 #include <service-evtmgr/ScreenLightControlMessage.hpp>
 #include <service-evtmgr/ServiceEventManagerName.hpp>
 
 #include <service-db/Settings.hpp>
 #include <db/SystemSettings.hpp>
 #include <module-utils/utility/Utils.hpp>
-#include <common/data/FrontlightUtils.hpp>
 
 namespace alarms
 {
     namespace
     {
+        constexpr std::string_view alarmFrontlightOff     = "0";
+        constexpr std::string_view prewakeupFrontlightOff = "0";
+
+        /* Reach and maintain target brightness 1 minute faster than
+         * the duration of the action. */
+        constexpr auto frontlightRampDurationOffset = std::chrono::minutes{1};
+        constexpr auto initialTargetDuration        = std::chrono::seconds{5};
+        constexpr auto initialZeroBrightnessValue   = 10;
+
         void validateBrightness(std::string &brightness)
         {
             constexpr std::string_view DefaultBrightness{"50.0"};
```

Comment:
I know it is not part of your PR but above you use `=` to `std::string_view`. Let's think about changing this ;)

Reply:
Done